test/NNPDE_tests_gpu.jl

```diff
@@ -43,12 +43,14 @@ dx = 0.25; dy= 0.25; dt = 0.25
 # Neural network
 chain = FastChain(FastDense(3,16,Flux.σ),FastDense(16,16,Flux.σ),FastDense(16,1))
```

Comment:
```suggestion
const gpuones = gpu(ones(1))
chain = FastChain(FastDense(3,16,Flux.σ),FastDense(16,16,Flux.σ),FastDense(16,1),(u,p)->gpuones .* u)
```

Reply:
This is a hacky way to "prove" it's running on the GPU, because multiplying by an array of ones on the GPU would fail if the `u` is on the CPU.